Update harfanglab.md (#559)
SEKOIA-IO/documentation: docs/xdr/features/collect/integrations/endpoint/harfanglab.md

## Configure

**Pre-requisite**
Harfang must setup openssl to the server (then Protocol `TCP/SSL` will be available)

**Steps**

> First your need to navigate to `Personal Settings`,  and in the `Api token` get your token or generate a new one.

Reviewer: ```suggestion
Go to `Personal Settings` > `API Key` and get your token or generate a new one.
```

> Then you need to navigate to `Administration > Configuration`, and switch to the `Connectors` tab.

Reviewer: ```suggestion
Then you need to navigate to `Administration` > `Configuration`, and switch to the `Connectors` tab.
```
